src/dodal/devices/smargon.py

```diff
@@ -64,6 +64,8 @@ class Smargon(MotorBundle):
 
     stub_offsets: StubOffsets = Cpt(StubOffsets, "")
 
+    disabled: EpicsSignal = Cpt(EpicsSignal, "DISABLED")
```

Comment:
```suggestion
    disabled = Cpt(EpicsSignal, "DISABLED")
```
Like the changes you made in https://github.com/DiamondLightSource/dodal/pull/266/files, we should do this here too